pkg/precompiles.sty

```diff
@@ -143,42 +168,53 @@
 
 \newcommand{\callCOneMembershipWCP}             {\texttt{callToC1MembershipWCP}}
 
-\newcommand{\callToCOneMembershipWCP}[6]{
-    \callCOneMembershipWCP_{#1}
+\newcommand{\callToCOneMembershipWCP}[1]{
+    \setkeys[ECDATA]{var}{#1}
+    \callCOneMembershipWCP_{\cmdECDATA@var@anchorRow}
     \left[ \begin{array}{ll}
-            \utt{Rel. row offset:}        & #2       \\
-            \utt{First argument (high):}  & \col{#3} \\
-            \utt{First argument (low):}   & \col{#4} \\
-            \utt{Second argument (high):} & \col{#5} \\
-            \utt{Second argument (low):}  & \col{#6} \\
+            \utt{Rel. row offset:}        & \cmdECDATA@var@relOffset        \\
+            \utt{First argument (high):}  & \col{\cmdECDATA@var@argOneHi}   \\
+            \utt{First argument (low):}   & \col{\cmdECDATA@var@argOneLo}   \\
+            \utt{Second argument (high):} & \col{\cmdECDATA@var@argTwoHi}   \\
+            \utt{Second argument (low):}  & \col{\cmdECDATA@var@argTwoLo}   \\
+            \utt{Third argument (high):}  & \col{\cmdECDATA@var@argThreeHi} \\
+            \utt{Third argument (low):}   & \col{\cmdECDATA@var@argThreeLo} \\
+            \utt{Fourth argument (high):} & \col{\cmdECDATA@var@argFourHi}  \\
+            \utt{Fourth argument (low):}  & \col{\cmdECDATA@var@argFourLo}  \\
         \end{array} \right]}
 
 \newcommand{\callCOneMembershipEXT}             {\texttt{callToC1MembershipEXT}}
 
-\newcommand{\callToCOneMembershipEXT}[6]{
-    \callCOneMembershipEXT_{#1}
+\newcommand{\callToCOneMembershipEXT}[1]{
+    \setkeys[ECDATA]{var}{#1}
+    \callCOneMembershipEXT_{\cmdECDATA@var@anchorRow}
     \left[ \begin{array}{ll}
-            \utt{Rel. row offset:}        & #2       \\
-            \utt{First argument (high):}  & \col{#3} \\
-            \utt{First argument (low):}   & \col{#4} \\
-            \utt{Second argument (high):} & \col{#5} \\
-            \utt{Second argument (low):}  & \col{#6} \\
+            \utt{Rel. row offset:}        & \cmdECDATA@var@relOffset        \\
+            \utt{First argument (high):}  & \col{\cmdECDATA@var@argOneHi}   \\
+            \utt{First argument (low):}   & \col{\cmdECDATA@var@argOneLo}   \\
+            \utt{Second argument (high):} & \col{\cmdECDATA@var@argTwoHi}   \\
+            \utt{Second argument (low):}  & \col{\cmdECDATA@var@argTwoLo}   \\
+            \utt{Third argument (high):}  & \col{\cmdECDATA@var@argThreeHi} \\
+            \utt{Third argument (low):}   & \col{\cmdECDATA@var@argThreeLo} \\
+            \utt{Fourth argument (high):} & \col{\cmdECDATA@var@argFourHi}  \\
+            \utt{Fourth argument (low):}  & \col{\cmdECDATA@var@argFourLo}  \\
```

Comment:
I don't think we need the `\col{...}` stuff here, do we ? All you give it are local shorthands `\locXxx` which are already under `\col{...}`.

Reply:
Also it would be nice to have more expressive names for the arguments, e.g.
```latex
\utt{x coord. hi}  & \col{\cmdECDATA@var@xHi}   \\
\utt{x coord. lo}  & \col{\cmdECDATA@var@xLo}   \\
\utt{y coord. hi}  & \col{\cmdECDATA@var@yHi}   \\
\utt{y coord. lo}  & \col{\cmdECDATA@var@yLo}   \\
\utt{LHS coord. hi}  & \col{\cmdECDATA@var@ySquareHi}   \\
\utt{LHS coord. lo}  & \col{\cmdECDATA@var@ySquareLo}   \\
\utt{RHS coord. hi}  & \col{\cmdECDATA@var@xCubePlusThreeHi}   \\
\utt{RHS coord. lo}  & \col{\cmdECDATA@var@xCubePlusThreeLo}   \\
```
with some explanation that the LHS and RHS refer to those of $y^2 = x^3 + 3$ or whatever the equation is.